docker/tds/docker-compose.yml

```diff
@@ -1,5 +1,20 @@
-version:  '2'
+version:  '2.1'
 services:
+  configuration-service:
+    image: fwsbac/configuration-service
+    ports:
+      - "32844:8888"
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:8888/health.json"]
+      interval: 30s
+      timeout: 10s
+      retries: 5
+    environment:
+      # Define these in your host machine environment
+      - CONFIG_SERVICE_REPO
+      - GIT_USER
+      - GIT_PASSWORD
+      - ENCRYPT_KEY
```

Comment:
This defines some pass-through environment variables that will be copied to the container from the host machine's environment.  I'll add standard values to the wiki.